docs/index.yml

```diff
@@ -199,7 +178,7 @@ additionalContent:
             summary: Learn how to create a data model with Azure data studio. Create indexes, tables and use data types to help develop your applications.
             url: /training/modules/design-a-data-model-with-ads/
 
-    - title: Cloud Solution Architecture for Linux
+    - title: Cloud Solution Architecture for Linux (REMOVE ME)
```

Comment:
I'm interested to look at where there may be a disconnect on this content for the audience visiting this page.... is this content just too advanced for our primary customer audience on this page? Do we need better explanations (at a Beginner-level)? How are the customers visiting this content discovering it?